### DISCUSSION TITLE
Forcing colors when writing to file

### DISCUSSION THREAD

**Discussion by rasmus-kirk** [obfusk/jiten#138] [Q&A] [answered]
Hi, I am using jiten as a terminal program for learning Japanese, recently I wanted to get a "word of the day" in my terminal. So I run a cron job every 10 minutes that checks if the date has changed and if it has, it updates the word. This does work as intended but I ran into issues when I wanted to also save the color to a file. I tried using both `unbuffer` and `script` to fool jiten into thinking that I was piping into an interactive shell. But this gave issues when it was run from a cron job as there is no terminal width, and jiten produced an error. 

So, can I force color somehow when outputting to a file?

**Comment by obfusk** [comment, marked as answer]
As `jiten --help` will tell you, you can use `--colour` (before the subcommand) :)

```bash
$ jiten --colour jmdict +random
```

**Reply by obfusk** [reply]
> jiten produced an error

That's odd. What error?

**Reply by rasmus-kirk** [reply]
Fuck, I can't believe I missed that, my bad. The errors were good, they complained that the terminal width was too low (which makes sense), since I ran this without an interactive shell and without writing directly to a file. It was just me doing it in a bad way. 

Thanks for the help!
